### PR TITLE
Serializing EagerTensors in model.save()

### PR DESCRIPTION
**Pull request by stratomaster31**
* This commit fixes problems when saving tf.keras.models coming from tensorflow 1.x
* If there are any EagerTensors in the graph, they are converted to numpy objects that can be serialized
* This fix has been tested and it works

### CODE REVIEW

File: tensorflow/python/util/serialization.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 from tensorflow.python.framework import dtypes
 from tensorflow.python.framework import tensor_shape
 from tensorflow.python.util.compat import collections_abc
-
+import tensorflow as tf
```
Comment (bhack) [inline tensorflow/python/util/serialization.py:27]: Please use `from tensorflow.python` like the other imports

File: tensorflow/python/util/serialization.py [hunk 2]
```diff
@@ -42,6 +42,10 @@ def get_json_type(obj):
   # e.g. optimizer, layer
   if hasattr(obj, 'get_config'):
     return {'class_name': obj.__class__.__name__, 'config': obj.get_config()}
+  
+  # Serialize EagerTensors (fix for saving 1.x Keras Models)
+  if isinstance(obj, tf.python.framework.ops.EagerTensor):
```
Comment (bhack) [inline tensorflow/python/util/serialization.py:47]: You need to create a new test for this feature at https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/tests/serialization_util_test.py

**Review by bhack — CHANGES_REQUESTED** [review]
It requires a test.

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]
Please add test and follow review suggestions from @bhack.

**Comment by gbaned** [comment]
@stratomaster31  Can you please check @mihaimaruseac's comments and keep us posted ? Thanks!

**Comment by gbaned** [comment]
@stratomaster31 Any update on this PR? Please. Thanks!

**Comment by tensorflowbutler** [comment]
It has been 24 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by gbaned** [comment]
@stratomaster31 Any update on this PR? Please. Thanks!

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!
